### PR TITLE
Refactoring the algorithm for calculating CPU usage 

### PR DESCRIPTION
**Pull request by TheBestLL**
refactor(process): Refactoring the algorithm for calculating CPU usage

##https://github.com/shirou/gopsutil/issues/1690

### CODE REVIEW

File: process/process.go [hunk 1]
```diff
@@ -325,7 +325,11 @@ func calculatePercent(t1, t2 *cpu.TimesStat, delta float64, numcpu int) float64
 	if delta == 0 {
 		return 0
 	}
-	delta_proc := t2.Total() - t1.Total()
+	//https://github1s.com/giampaolo/psutil/blob/c034e6692cf736b5e87d14418a8153bb03f6cf42/psutil/__init__.py#L1064
```
Comment (shirou) [inline process/process.go:328]:
```suggestion
	// https://github.com/giampaolo/psutil/blob/c034e6692cf736b5e87d14418a8153bb03f6cf42/psutil/__init__.py#L1064
```

**Review by shirou — CHANGES_REQUESTED** [review]
Good catch! I want to merge but could you fix the small comment?

File: process/process.go (hunk 1 above)
Comment (TheBestLL) [reply process/process.go:328]: Repair the reference URL.

**Comment by TheBestLL** [comment]
Of course, thank you very much.

**Review by shirou — APPROVED** [review]

**Pull request merged**